pkg/sharing/replicator.go

```diff
@@ -208,9 +209,17 @@ func (s *Sharing) replicationID(m *Member) (string, error) {
 	return "", ErrMemberNotFound
 }
 
-// Changes is a map of "doctype/docid" -> [revisions]
-// It's the format for the request body of our _revs_diff
-type Changes map[string][]string
+// Changed is a map of "doctype/docid" -> [revisions]
+type Changed map[string][]string
+
+// Removed is a set of "doctype/docid"
+type Removed map[string]struct{}
```

Comment:
Why a struct and not a bool?

Reply:
It's an optimization. A bool takes one byte, but an empty struct is empty and takes 0 bytes.